From 9c874ee6f595906faf59f891f20492530888804b Mon Sep 17 00:00:00 2001
From: Ray Strode <rstrode@redhat.com>
Date: Mon, 24 Oct 2011 16:45:46 -0400
Subject: [PATCH] worker: don't block SIGTERM

If the slave tells us to go away, we should go away,
not wait a PAM module decides to let us get back to the
main loop.
---
 daemon/session-worker-main.c |    1 -
 1 files changed, 0 insertions(+), 1 deletions(-)

diff --git a/daemon/session-worker-main.c b/daemon/session-worker-main.c
index 9d40b8d..42fcd52 100644
--- a/daemon/session-worker-main.c
+++ b/daemon/session-worker-main.c
@@ -188,7 +188,6 @@ main (int    argc,
         gdm_signal_handler_set_fatal_func (signal_handler,
                                            (GDestroyNotify)g_main_loop_quit,
                                            main_loop);
-        gdm_signal_handler_add (signal_handler, SIGTERM, signal_cb, NULL);
         gdm_signal_handler_add (signal_handler, SIGINT, signal_cb, NULL);
         gdm_signal_handler_add (signal_handler, SIGILL, signal_cb, NULL);
         gdm_signal_handler_add (signal_handler, SIGBUS, signal_cb, NULL);
-- 
1.7.8.1

